### ISSUE TITLE
www.tueq.com

### ISSUE THREAD

**Issue by salman01** [GavickPro/Magazine-Free-Joomla-Template#11]
dear sir,
i am using gavick free magazine template and i think there is a problem with the toolbar ,

the toolbar is collaiding onto the logo
and when i use chrome the logo goes into the toolbar,

can you please log into my website and check this problem 
www.tueq.com

thanks alot

**Comment by aml987** [comment]
edit this css file: http://www.tueq.com/templates/gk_magazine/css/template.css
in line 113 , 
edit the line to `padding: 0 60px;`

---

can you give me the URL of slide show that appears on your site?

**Comment by salman01** [comment]
hi aml987, stll didnt work
i did what you said without luck :(

sure its called Slideshow CK

**Comment by aml987** [comment]
Press CTRL+F5 on your page after editing, and upload a picture here of where is your problem exactly.
if it doesn't work: wait @bkrztuk  to reply you

**Comment by bkrztuk** [comment]
I swą your website but there is no magazine template but game magazine - our commerial product. If you still have a problem please write on our official Support forum

**Comment by salman01** [comment]
Yeah its game magazine that i bought recently and i have problem with that am trying to change the background picture without luck can you please help me with that @bkrztuk

**Comment by bkrztuk** [comment]
If you have commercial template please use our support forum gavick.com/forum where you get help much faster that in repository for free templates.
